code-challenges.test.js

```diff
@@ -26,42 +26,110 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
-
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
-
+// describe("divisibleBy3", () => {
+//     it("Takes in an object and returns if divisible by three.", () => {
+//       const object1 = { number: 15 }
+//       const object2 = { number: 0 }
+//       const object3 = { number: -7 }
+//       expect(divisibleBy3(object1.number)).toEqual("is divisible by three")
+//       expect(divisibleBy3(object2.number)).toEqual("is divisible by three")
+//       expect(divisibleBy3(object3.number)).toEqual("is not divisible by three")
+//     })
+//   }) //● divisibleBy3 › Takes in an object and returns if divisible by three.
+  
 // b) Create the function that makes the test pass.
-
+const divisibleBy3 = (object) => {
```

Comment:
Refer to above comment about the argument and expected output for this challenge.